Build fails with `assimp` feature

Building fails if `assimp` feature is enabled. (Sorry if I have misunderstood something)

Compiler diagnostic:

```bash
$ cargo install urdf-viz --features assimp
# ...
   Compiling urdf-viz v0.43.1
error[E0308]: mismatched types
   --> /home/pi/.cargo/registry/src/index.crates.io-6f17d22bba15001f/urdf-viz-0.43.1/src/assimp_utils.rs:71:13
    |
69  |         match assimp_sys::aiGetMaterialColor(
    |               ------------------------------ arguments to this function are incorrect
70  |             mat,
71  |             color_type.as_ptr() as *const i8,
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `*const u8`, found `*const i8`
    |
    = note: expected raw pointer `*const u8`
               found raw pointer `*const i8`
note: function defined here
   --> /home/pi/.cargo/registry/src/index.crates.io-6f17d22bba15001f/assimp-sys-0.3.1/src/material.rs:146:12
    |
146 |     pub fn aiGetMaterialColor(
    |            ^^^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `urdf-viz` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `urdf-viz v0.43.1`, intermediate artifacts can be found at `/tmp/cargo-installake7O1`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

- OS: Ubuntu 22.04
- `rustc --version`: `rustc 1.72.0 (5680fa18f 2023-08-23)`

--- a/src/assimp_utils.rs
+++ b/src/assimp_utils.rs
@@ -66,9 +66,10 @@ pub fn assimp_material_color(
     };
     let mat = &(**material) as *const assimp_sys::AiMaterial;
     unsafe {
+        use std::os::raw::c_char;
         match assimp_sys::aiGetMaterialColor(
             mat,
-            color_type.as_ptr() as *const i8,
+            color_type.as_ptr() as *const c_char,
             0,
             0,
             &mut assimp_color,

